arbiter-core/src/lib.rs

```diff
@@ -1,3 +1,22 @@
+//! `arbiter-core` is designed to facilitate agent-based simulations of Ethereum
```

Comment:
Thoughts on adding some ASCII art here? I kinda like this
```

           _____  ____ _____ _______ ______ _____  
     /\   |  __ \|  _ \_   _|__   __|  ____|  __ \ 
    /  \  | |__) | |_) || |    | |  | |__  | |__) |
   / /\ \ |  _  /|  _ < | |    | |  |  __| |  _  / 
  / ____ \| | \ \| |_) || |_   | |  | |____| | \ \ 
 /_/    \_\_|  \_\____/_____|  |_|  |______|_|  \_\
                                                   
                                                   
```

But i know there are lots of versions to be explored [here](https://patorjk.com/software/taag/#p=testall&v=0&f=Slant&t=ARBITER)

Reply:
I like ascii art so yes, I am down.

We should have some artwork for when we run things too!